modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, chronyd, mountpoint hiding, sctp enablement, and more.
+* `ExtraManifestsConfigMap` - Extra manifests. The extra manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. To automatically merge the extra manifests into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`, set the `.spec.clusters.mergeDefaultMachineConfigs` to `true` in the `SiteConfig.yaml` file.
```

Comment:
I do not see this paragraph in any of the previews. Could you make sure it's there?

Reply:
@stevsmit I needed to update the previews, here it is: https://48093--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-deploying-a-site_ztp-deploying-disconnected